### ISSUE TITLE
Tutorial Refactors

### ISSUE THREAD

**Issue by CSSFrancis** [ePSIC-DLS/Hyperspy_Workshop_2024#1]
Just thinking about the changes from last year and what we should think about changing:

Dask/ Lazy Processing
----------------------
- [ ] Dask Distributed (How to use it and what does everything mean)
- [ ] File Formats --> Zarr vs HDF5
- [ ] Lazy Marker Plotting?

Data Visualization
------------------
- [ ] Plotting using (new) Markers
- [ ] https://github.com/hyperspy/hyperspy/pull/3333
- [ ] https://github.com/hyperspy/hyperspy/pull/3343

EELS
-----
- [ ] New edges for fitting!

Pyxem
-------
- [ ] Redo Tutorials based on 1.0.0 release 

Rosettasciio
------------
- [ ] General session on how to use Rosettasciio separately.  @smribet maybe in the py4dstem session?

Are there any other additions? Feel free to edit my post if you want.

**Comment by smribet** [comment]
Hi Carter, 

I'd be interested to chat more about Rosettasciio either during the workshop or another time. I don't think it would fit in the py4DSTEM section, since we have a lot of other material. Data IO is a pretty general topic. If it is included, perhaps it would make senes to go towards the beginning or end of the workshop as part of a larger discussion? 

~Stephanie

**Comment by CSSFrancis** [comment]
> I'd be interested to chat more about Rosettasciio either during the workshop or another time. 

Sure, send me an email, and we can set up a meeting anytime!

> I don't think it would fit in the py4DSTEM section, since we have a lot of other material. Data IO is a pretty general topic. If it is included, perhaps it would make senes to go towards the beginning or end of the workshop as part of a larger discussion?

I think that in the hyperspy section, we can quite easily talk about using Rosettasciio.  For the py4dstem section, it would be nice to show something like loading a `.mib`  or .`hspy` dataset into py4DSTEM using rosettasciio quickly but not go into the details. Mapping the metadata from one to the other/ having a working code example so that people have it if they need it seems useful.

If you would like, I can write up an example showing this. I think a common workflow at Diamond is to load the data using hyperspy/pyxem and then transfer it to py4dstem for certain processing (@M0hsend you could correct me on that!) so it would be usefully regaurdless.

**Comment by M0hsend** [comment]
@CSSFrancis That is true. You can see an example in this notebook [here](https://github.com/ePSIC-DLS/User-Notebooks/blob/master/ePSIC_Standard_Notebooks/template_BraggAnalysis.ipynb).

**Comment by magnunor** [comment]
@CSSFrancis, I added the same dask distributed as last year: https://github.com/ePSIC-DLS/Hyperspy_Workshop_2024/pull/6

I have not tried running that one specifically (I tested the lazy big data one I ran last year).

**Comment by CSSFrancis** [comment]
@magnunor that should be good. I can test it a little later (probably will end up being tomorrow at this point)!
